src/coreclr/jit/jit.h

```diff
@@ -529,6 +529,8 @@ const bool dspGCtbls = true;
     if (JitTls::GetCompiler()->verbose)                                                                                \
         JitTls::GetCompiler()->fgTableDispBasicBlock(b);
 #define VERBOSE JitTls::GetCompiler()->verbose
+#define ISMETHOD(name) !strcmp(JitTls::GetCompiler()->info.compMethodName, name)
+#define ISMETHODHASH(hash) !strcmp(JitTls::GetCompiler()->info.compMethodHash, hash)
```

Comment:
Does the hash version work? Seems like it would need to compare unsigned ints and use `info.compMethodHash()`.

Also you might want to update these to check the root compiler so they don't trigger for inlinees. So something like:
```suggestion
#define ISMETHODHASH(hash) (JitTls::GetCompiler()->impInlineRoot()->info.compMethodHash() == hash)
```

Reply:
oops